geom/FieldBLAS_IntrinsicFunctions.F90

```diff
@@ -0,0 +1,60 @@
+module FieldBLASIntrinicFunctions
```

Comment:
This file is not being compiled currently.  Just wanted to be sure @darianboggs is aware.

Reply:
Correct. Not in `CMakeLists`. I'll leave the extra files for now.